tests/test_linear_objectives.py

```diff
@@ -186,7 +180,7 @@ def test_fixed_mode_solve():
     # and check that the mode stayed fix
     L = 1
     M = 1
-    eq = desc.examples.get("DSHAPE")
+    eq = get("DSHAPE")
```

Comment:
personally I prefer the explicit `desc.examples.get`, since `get` by itself is pretty generic, and there are other methods in python with the same name